docs/deployment/content/remote-local-setup.yaml

```diff
@@ -64,7 +64,7 @@ spec:
           tmux split-window \; select-pane -T kind     ; sleep 1; tmux send "make kind-up # Set up KinD cluster (Garden and Seed)" \; select-layout even-vertical
           tmux split-window \; select-pane -T gardener ; sleep 1; tmux send "make gardener-up # Set up Gardener"\; select-layout even-vertical
           tmux split-window \; select-pane -T shoot    ; sleep 1; tmux send "kubectl apply -f example/provider-local/shoot.yaml # Create a new shoot cluster "\; select-layout even-vertical
-          tmux split-window \; select-pane -T config   ; sleep 1; tmux send "kubectl -n garden-local get secret local.kubeconfig -o jsonpath={.data.kubeconfig} | base64 -d > /tmp/kubeconfig-shoot-local.yaml" \; select-layout even-vertical
+          tmux split-window \; select-pane -T config   ; sleep 1; tmux send "./hack/usage/generate-admin-kubeconf.sh > admin-kubeconf.yaml; export KUBECONFIG=/root/gardener/admin-kubeconf.yaml # Get Kubeconfig for Shoot" \; select-layout even-vertical
```

Comment:
I would prefer to save the kubeconfig in a folder which does not make the git repo directory dirty.
You could keep `/tmp/kubeconfig-shoot-local.yaml` as destination, which is also in `KUBECONFIG` env already (see line 41).

Reply:
Thank you for the hint 😊
I have implemented it in [cb933ea](https://github.com/gardener/gardener/pull/9023/commits/cb933eaa01165b3b7b08c75c5fa1c391be49abeb)